src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+public class ShooterSubsystem extends SubsystemBase {
+    private static TalonFX top;
+    private static TalonFX bottom;
+    private final VoltageOut voltageOut = new VoltageOut(0).withEnableFOC(true);
```

Comment:
Will this be used anywhere else? Only reference I can find is on this line at the moment.